service/model/actions.go

```diff
@@ -185,6 +186,34 @@ func assertionHeaders(w http.ResponseWriter, user datastore.User, apiCall bool,
 		return
 	}
 
+	// create a signed model assertion
+	assertionHeaders, keypair, err := datastore.CreateModelAssertionHeaders(model)
+	if err != nil {
+		log.Println(err)
+		response.FormatStandardResponse(false, "error-signing-assertions", "", err.Error(), w)
+		return
+	}
+
+	signedAssertion, err := datastore.Environ.KeypairDB.SignAssertion(asserts.ModelType,
+		assertionHeaders,
+		[]byte(""),
+		model.BrandID,
+		keypair.KeyID,
+		keypair.SealedKey)
+	if err != nil {
+		log.Println(err)
+		response.FormatStandardResponse(false, "error-signing-assertions", "", err.Error(), w)
+		return
+	}
+
+	// store a signed model assertion in the database
+	err = datastore.Environ.DB.UpsertSignedModelAssert(model.ID, signedAssertion)
```

Comment:
if I understand correctly here we will just replace whatever we have in the db with a new signed assertion, for that to respect assertion rules we really need to check if have something already in the db and sign assertion with current revision + 1 as the newer replacement

Reply:
Yes, this is correct. I can implement this check